build/yamls/antrea-aks.yml

```diff
@@ -3296,6 +3296,9 @@ spec:
       initContainers:
       - command:
         - install_cni_chaining
+        env:
+        - name: SKIP_CNI_BINARIES
```

Comment:
Could you provide a sample about how to set this variable? I didn't see you mentioned it anywhere. 
I remember kustomize will remove all comments from yaml @tnqn any suggestion about where to put the sample information?

Reply:
can I add the comment in the antrea.yml file

Reply:
It should be used by distros and end users won't care about it. So perhaps adding comment to the base yaml is enough, I think it's fine to not have it in the generated yaml.

Reply:
yes sure would do that